client/console_conf.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+import "time"
+
+// ConsoleConfStartResponse is the response for starting the console-conf
+// routine.
+type ConsoleConfStartResponse struct {
+	ActiveAutoRefreshChanges []string `json:"active-auto-refreshes,omitempty"`
+	ActiveAutoRefreshSnaps   []string `json:"active-auto-refresh-snaps,omitempty"`
+}
+
+// ConsoleConfStart starts the console-conf start routine.
+func (client *Client) ConsoleConfStart() ([]string, []string, error) {
```

Comment:
this should be called InternalConsoleConfRestart I think. Another approach would be to have an Internal helper a bit like Debug (but the details would need to be different)

Reply:
I will stick with InternalConsoleConfRestart for now, I agree Internal might be nice but would be more details to work out